libs/documentation/src/lib/storybook/autocomplete/autocomplete-checkbox/autocomplete-checkbox.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { SdsAutocompleteModule } from '@gsa-sam/components';
+import { FormsModule } from '@angular/forms';
+import { AutocompleteCheckbox } from './autocomplete-checkbox.component';
+import { NgxBootstrapIconsModule, square } from 'ngx-bootstrap-icons';
+import { add, IconModule } from '@gsa-sam/ngx-uswds-icons';
+
+@NgModule({
+  declarations: [AutocompleteCheckbox],
+  imports: [
+    CommonModule,
+    FormsModule,
+    SdsAutocompleteModule,
+    IconModule,
+    NgxBootstrapIconsModule.pick({ square, add }),
```

Comment:
Is importing the icons actually needed? This icons should be provided in the root and just be available here.

Reply:
no, this is extra, i removed it